### DISCUSSION TITLE
What Format are Downloaded Songs Stored in?

### DISCUSSION THREAD

**Discussion by Angy-Kitty** [z-huang/music#993] [Q&A]
I am looking at the Android/data/com.zionhuang.music/files/media/ on my pc, and the file type is showing up as 'unkown'. I would like to know how the files are stored.

**Comment by Angy-Kitty** [comment]
It would also be nice to be able to export the files as MP3 or WAV

**Comment by z-huang** [comment]
The songs are downloaded in chunks, so you probably can't open the file directly. Actually, I'm considering giving an option to export songs to mp3/wav, but the priority to do it isn't high.
